internal/attribute/attribute_test.go

```diff
@@ -94,3 +94,27 @@ func TestSliceValue(t *testing.T) {
 		})
 	}
 }
+
+func BenchmarkBoolSliceValue(b *testing.B) {
+	for n := 0; n < b.N; n++ {
+		_ = BoolSliceValue([]bool{true, false, true, false})
```

Comment:
Assignment to empty can be optimized away by the compiler. This should be assigned to a var out of scope.